packages/dds/tree/src/test/objectForest.spec.ts

```diff
@@ -66,9 +66,9 @@ function testForest(
                     assert(!isNeverField(defaultSchemaPolicy, schema, rootFieldSchema));
 
                     // TODO: use new JsonCursor directly when ready instead of converting.
```

Comment:
This comment can be removed now, right?

Reply:
This is left over from not having JsonCursor in the new API, I'll make another little PR for that cleanup as its technically a separate issue.